md/product-versioning.md

```diff
@@ -0,0 +1,51 @@
+# Bonita Versioning
+Discover how Bonita manages its product and artifacts versions and how it can impact your projects
+
+## Product Versions
+
+Bonita follows the usual semantic versioning X.Y.z:
+* X is the major version number. It changes when non-backward compatible changes are implemented.
+* Y is the minor version number. It version changes when backward compatible features are added. A minor version is released every six months to improve continuously Bonita.
+* z is the maintenance version number. The maintenance versions contain fixes. Maintenance versions are released every month between two minor versions for the current minor version. These versions are cross-compatible by default for the same minor version.
+
+## Artifact version
+
+In addition to the product version, most artifacts have a model version. The development suite (Studio and UID) uses this model version to determine if the corresponding artifact is compatible.
```

Comment:
In the external docs, we always say "UI Designer", and never use "UID"
"... to determine if the artifacts are compatible."